src/containers/Application/ElementsArea.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 import { Button, Container, Grid, Header, Label, Segment } from 'semantic-ui-react'
 import { ApplicationViewWrapper } from '../../elementPlugins'
-import useLoadElements from '../../utils/hooks/useLoadElements'
+import useLoadElements from '../../utils/hooks/useLoadElementsOfSection'
```

Comment:
```suggestion
import useLoadElementsOfSection from '../../utils/hooks/useLoadElementsOfSection'
```

And rename where is in use. It makes it simpler to use the same name as the file.

Reply:
Yes, good catch. This is what happens when you use the "Rename Symbol" feature in VSCode. Annoying.